src/pieces/commandManager.ts

```diff
@@ -85,6 +85,10 @@ async function runCommand(msg: Message) {
 
 	const command = getCommand(msg.client, commandName);
 	if (!command || command.enabled === false) return;
+	if (msg.channel.id !== CHANNELS.SAGE && command.restricted) {
```

Comment:
Should we allow Admin to use Sage commands everywhere? (Genuine question, not requesting a change here lol)

Reply:
I don't see why not, but I also don't see why we'd need to use the restricted commands outside of Sage's Place. I think things are fine as-is for now unless we see fit to change it later.